docs/design/disable-add-in-commands.md

```diff
@@ -71,24 +71,31 @@ By default, any Add-in Command is enabled when the Office application launches.
 
 The essential steps to changing the enabled status of an Add-in Command are:
 
-1. Create a [RibbonUpdaterData](/javascript/api/office/office.ribbonupdaterdata) object that (1) specifies the command, and its parent tab, by their IDs as specified in the manifest; and (2) specifies the enabled or disabled state of the command.
+1. Create a [RibbonUpdaterData](/javascript/api/office/office.ribbonupdaterdata) object that (1) specifies the command, and its parent group and tab, by their IDs as specified in the manifest; and (2) specifies the enabled or disabled state of the command.
```

Comment:
Would it still be accurate if one of the instances of 'specify' is changed? Seeing 'specify' so many times in a row makes understanding the sentence difficult for me. 
```suggestion
1. Create a [RibbonUpdaterData](/javascript/api/office/office.ribbonupdaterdata) object that (1) specifies the command, and its parent group and tab, by their IDs as declared in the manifest; and (2) specifies the enabled or disabled state of the command.
```